projects/metis/src/app/dashboard/filter-ops/filter-ops-conf.ts

```diff
@@ -12,6 +12,125 @@ import { RenameWorkflowPipe } from '../../_translate';
 
 const today = new Date().toISOString().split('T')[0];
 
+/**
+ * setDateFrom
+ *
+ * Function to invoke when the date-from is set
+ *
+ * @param { CanHaveError } - cmp - the component
+ * @param { HTMLInputElement } - el - the native element
+ * @param { HTMLElement[] } - opElements - elements related in the config group
```

Comment:
According to my IDE this is not valid JSDoc: the first hyphen needs to be removed in all 3 lines.